app/controllers/idv/inherited_proofing_cancellations_controller.rb

```diff
@@ -8,9 +8,7 @@ class InheritedProofingCancellationsController < ApplicationController
     before_action :confirm_idv_needed
```

Comment:
Hey @jscodefix don't forget the [sessions controller](https://github.com/18F/identity-idp/blob/main/app/controllers/idv/sessions_controller.rb) (https://cm-jira.usa.gov/browse/LG-7987); this gets called as part of the cancellations process when "Start over" is clicked

Reply:
Deferring to later implementation; needs further research.